proposals/dictionary-expressions.md

```diff
@@ -6,10 +6,10 @@
 
 Several dictionary-like types can created without external BCL support.  These types are:
 
-1. Concrete dictionary-like types, containing an indexer `TValue this[TKey] { get; }`, like `Dictionary<TKey, TValue>`, `ConcurrentDictionary<TKey, TValue>`, `ImmutableDictionary<TKey, TValue>`, and `FrozenDictionary<TKey, TValue>`.
```

Comment:
i realized this was wrong.  ID and FD shouldn't be in this list.  They should be in the list we need runtime 'create method' support for.